packages/client/tests/e2e/publish-extensions/simple-ext/package.json

```diff
@@ -7,7 +7,7 @@
     "build": "tsc"
   },
   "peerDependencies": {
-    "@prisma/client": "/tmp/prisma-client-0.0.0.tgz"
+    "@prisma/client": "workspace:/tmp/prisma-client-0.0.0.tgz"
```

Comment:
ℹ️ This is the other fix: `pnpm` v10 changed validation for peerDependencies and does not allow a plain path anymore.